content/en/opentelemetry/compatibility.md

```diff
@@ -0,0 +1,95 @@
+| Database Monitoring (DBM)                     |                                    |                                    | {{< X >}}                           | {{< X >}}<br>(Datadog SDK only)                      | {{< X >}}                          |
+| Data Streams Monitoring (DSM)                 |                                    |                                    | {{< X >}}                           | {{< X >}}<br>(Datadog SDK only)                      | {{< X >}}                          |
+| Live Processes                                |                                    | {{< X >}}                          | {{< X >}}                           | {{< X >}}<br>(Datadog SDK only)                      | {{< X >}}                          |
+| Universal Service Monitoring (USM)            |                                    |                                    | {{< X >}}                           | {{< X >}}<br>(Datadog SDK only)                      | {{< X >}}                          |
+| Cloud SIEM                                    | {{< X >}}                          | {{< X >}}                          | {{< X >}}                           | {{< X >}}                                            | {{< X >}}                          |
+| Correlated Traces, Metrics, Logs              | {{< X >}}                          | {{< X >}}                          | {{< X >}}                           | {{< X >}}                                            | {{< X >}}                          |
+| Live Container Monitoring/Kubernetes Explorer |                                    | {{< X >}}                          | {{< X >}}                           | {{< X >}}                                            | {{< X >}}                          |
+| Network Performance Monitoring (NPM)          |                                    | {{< X >}}                          | {{< X >}}                           | {{< X >}}                                            | {{< X >}}                          |
+| Real User Monitoring (RUM)                    | {{< X >}}                          | {{< X >}}                          | {{< X >}}                           | {{< X >}}                                            | {{< X >}}                          |
+| Runtime Metrics                               | {{< X >}}<br>(Java, .NET, Go only) | {{< X >}}<br>(Java, .NET, Go only) | {{< X >}}                           | {{< X >}}<br>(Java, .NET, Go only)                   | {{< X >}}                          |
+| Source code integration                       | {{< X >}}                          | {{< X >}}                          | {{< X >}}                           | {{< X >}}<br>(Datadog SDK only)                      | {{< X >}}                          |
+
+## More details
+
+### Runtime Metrics
```

Comment:
```suggestion
### Runtime metrics
```

Vale is flagging this, but since this is referring to specific OTEL runtime metrics, it might be ok to leave capitalized, your call!